contracts/mock/MockLiquidatorWithCallback.sol

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: agpl-3.0
+pragma solidity 0.7.6;
+
+import {IERC20Ext} from '@kyber.network/utils-sc/contracts/IERC20Ext.sol';
+import {Utils} from '@kyber.network/utils-sc/contracts/Utils.sol';
+import {SafeERC20} from '@openzeppelin/contracts/token/ERC20/SafeERC20.sol';
+import {MockLiquidationStrategy} from './MockLiquidationStrategy.sol';
+
+contract MockLiquidatorWithCallback is Utils {
+  using SafeERC20 for IERC20Ext;
+
+  uint256 public transferBackAmount = 0;
+  bool public shouldTestReentrancy = false;
+
+  constructor( ) {}
+
+  receive() external payable {}
+
+  function setTransferBackAmount(uint256 amount) external {
+    transferBackAmount = amount;
+  }
+
+  function setTestReentrancy(bool shouldTest) external {
+    shouldTestReentrancy = shouldTest;
+  }
+
+  function liquidationCallback(
```

Comment:
Thinking about this again and it would be better if we can pass minReturn amount as params